tests/unit/ds-macro-test.js

```diff
@@ -37,3 +37,32 @@ test('DS macro', function(assert) {
 
   assert.deepEqual(relationships, ['user', 'car']);
 });
+
```

Comment:
You should probably define the class inside the body of the test itself, see the `@action` decorator test for an example